packages/@okta/vuepress-site/guides/sign-into-spa/section12/angular/samples.md

```diff
@@ -0,0 +1 @@
+We have multiple samples available on our [Angular Samples](https://github.com/okta/samples-js-angular) repo.
```

Comment:
We have multiple samples available in our [Angular Samples](https://github.com/okta/samples-js-angular) repo.